tests/standalone/ClickUsing.py

```diff
@@ -0,0 +1,16 @@
+import click
+
+# nuitka-skip-unless-imports: click
+
+@click.command()
+#@click.option("--count", default = 1, help = "Number of greetings.")
+#@click.option("--name", prompt = "Your name", help = "The person to greet.")
+
+def hello(count = 5, name = "Tabi"):
+	"""Simple program that greets NAME for a total of COUNT times."""
+	for _ in range(count):
+		click.echo("Hello, %s!" % name)
+
+
+if __name__ == '__main__':
+	hello()
```

Comment:
The way click should be tested is by passing some options as CLI args while invoking the program.
Example: `python ClickUsing.py --count=3`. Can you think of a way to simulate that behaviour here?

Our goal here should be to check if click can parse those args correctly and allocate correct values to the options.

Reply:
I'll do this and commit.

Reply:
I think the next commit solves this. Please check if it does.